recipes/flac/meta.yaml

```diff
@@ -0,0 +1,50 @@
+{% set name = "flac" %}
+{% set version = "1.3.1" %}
+{% set sha256 = "e0c23c92af0aac830683a6ab6784dd0c11aaf04249709b9d600675b3b2441c3e" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/xiph/{{ name }}/archive/{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  features:
+    - vc9  # [win and py27]
+    - vc10  # [win and py34]
+    - vc14  # [win and py35]
+
+requirements:
+  build:
+    - python  # [win]
+    - cmake  # [win]
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+    - pkgconfig  # [unix]
+    - autoconf  # [unix]
+    - automake  # [unix]
+    - libtool  # [unix]
+    - gettext  # [unix]
+    - ogg
+  run:
+    - vc 9  # [win and py27]
+    - vc 10  # [win and py34]
+    - vc 14  # [win and py35]
+    - ogg
+
```

Comment:
Yes, I'll add the file check tests again, but didn't get to that yet. Working on it now. Additionally, on Unix the make check command seems to do some functional tests of the library already.